dojo/fixtures/test_type.json

```diff
@@ -47,5 +47,12 @@
         },
         "model": "dojo.test_type",
         "pk": 7
+    },
+    {
+        "fields": {
+            "name": "Whispers"
+        },
+        "model": "dojo.test_type",
+        "pk": 8
     }
```

Comment:
this is not needed, this file is still here for historical reasons

Reply:
This was automatically added by the cookiecutter..